frontend/src/scenes/sessionRecordings/SessionPlayerDrawer.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+
+import { Drawer } from 'lib/components/Drawer'
+import { SessionsPlay } from 'scenes/sessions/SessionsPlay'
+import { useValues } from 'kea'
+import { ArrowTopLeftOutlined } from 'lib/components/icons'
+import { featureFlagLogic } from 'lib/logic/featureFlagLogic'
+import { FEATURE_FLAGS } from 'lib/constants'
+
+interface SessionPlayerDrawerProps {
+    isPersonPage?: boolean
+    onClose: () => void
+}
+
+export function SessionPlayerDrawer({ isPersonPage = false, onClose }: SessionPlayerDrawerProps): JSX.Element {
+    const { featureFlags } = useValues(featureFlagLogic)
+    return (
+        <Drawer destroyOnClose visible width="100%" onClose={onClose}>
+            <>
+                <a onClick={onClose}>
+                    <ArrowTopLeftOutlined /> Back to{' '}
+                    {isPersonPage
+                        ? 'persons'
+                        : featureFlags[FEATURE_FLAGS.REMOVE_SESSIONS]
+                        ? 'sessions recordings'
+                        : 'sessions'}
+                </a>
```

Comment:
An `<a>` tag without a `href` is a suspicious thing that can randomly break things. Could this just be a `div`, or a `button`, or something else? Ideally even, can you give this a href, so people could open this in a new tab.

Reply:
I tried the `href` approach, but it made closing the modal very slow because it led to a page refresh, so I moved to using a button. 

I Googled around a bit for a way to stop the refresh on clicking <a href="">, but I didn't find anything obvious. Am I missing something here?

Reply:
Ah, yes, we have a separate `Link` tag for this. You can also always do `<a href='whatever' onClick={e => {e.preventDefault(); doWhateverElseNoLinkWillBeClicked() }}`